.github/bin/dist-github-release.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Extract a release note from the given changelog file.
+# Note that this script is intended to be run by GitHub Actions.
+# shellcheck disable=SC2169
+set -e
+
+# shellcheck source=constants.sh
+. "$(dirname "$0")/constants.sh"
+
+if [ "$GITHUB_REPOSITORY" = "" ] | [ "$GITHUB_REF" = "" ]; then
+    echo "This script is intended to be run by GitHub Actions." >/dev/stderr
+    exit 1
+elif [ "${GITHUB_REF:0:10}" != "refs/tags/" ]; then
+    echo "This script is run for only tag push; being skipped..." >/dev/stderr
+    exit 0 # If it exits with non-zero other actions become cancelled
+fi
+
+tag="${GITHUB_REF#refs/*/}"
+
+if command -v apk; then
+    apk add --no-cache ca-certificates
```

Comment:
apk? isn't it ubuntu?

Reply:
@.@ I guess this line is for compatibility
It's from libplanet `dist-github-release.sh`